third_party/nvidia/backend/compiler.py

```diff
@@ -447,11 +447,9 @@ def make_ttgir(mod, metadata, opt, capability):
             nvidia.passes.ttnvgpuir.add_wsmaterialization(pm, capability)
             passes.common.add_licm(pm)
             passes.common.add_cse(pm)
-        else:
+        elif capability // 10 >= 8:
             passes.ttgpuir.add_pipeline(pm, opt.num_stages, opt.num_warps, opt.num_ctas, capability)
         nvidia.passes.ttnvgpuir.add_materialize_load_store(pm, opt.num_warps, capability)
-        if capability // 10 <= 8:
-            passes.ttgpuir.add_prefetch(pm)
```

Comment:
I think this means that the prefetch pass is basically dead code at this point, right @ptillet?
I was already wondering about this one. What was it supposed to do in the first place?